docs/docs/deploying-to-qovery.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Deploying to Qovery
+---
+
+## Introduction
+
+[Qovery](https://www.qovery.com) is a fully-managed Cloud platform where you can host your modern application (web, database, broker, storage).
+
+> Trusted by developers from great companies - Google, Red Hat, Elastic and more
+
+Qovery include the following:
+
+- Continuous, automatic builds & deploys from GitHub, Gitlab and Bitbucket.
+- [Custom domains](https://docs.qovery.com/services/network/dns).
+- Automatic TLS certificates through [Let's Encrypt](https://letsencrypt.org).
+- Microservices support.
+- Auto-scaling support.
+- Native HTTP/2 support.
+- [Isolated environment](https://docs.qovery.com/extending-qovery/branches-and-environments) per git branch.
+- Managed databases (PostgreSQL, MySQL, MongoDB, Elasticsearch, Redis, Memcached, Cassandra).
+- Managed brokers (RabbitMQ, Kafka).
+- Managed storage (S3).
+- Custom URL redirects and rewrites.
+
+## Prerequisites
+To get started, you'll need:
+
+- You have an account with GitHub, GitLab or Bitbucket.
+- You have completed the Quick Start or have a Gatsby website you are ready to deploy and share with the world.
+- You have a Qovery account. You can sign up at [start.qovery.com](https://start.qovery.com).
+
+## Deploy your application
+
+You can set up a Gatsby site on Qovery in five quick steps:
+
+**1/** Install the Qovery CLI
+Here is [how to install the Qovery CLI](https://docs.qovery.com/extending-qovery/cli) documentation
+
+**2/** Sign up
+
+```bash
+# Sign up with Github, Gitlab, Bitbucket
+$ qovery auth
+```
+
+**3/** Configure your project and generate a .qovery.yml
+
+```bash
+# generate the .qovery.yml at the root of your project directory
+$ qovery init
```

Comment:
While running this I found that there are a bunch of other permissions that need to be done and this command isn't instant. Mind adding this to the guide ? We aim to give the users a clear definition of what to expect while working with this. ![](https://res.cloudinary.com/ekwuno/image/upload/v1586089057/Screenshot_2020-04-05_at_12.56.27.png)

Reply:
Maybe add a link to the part of the Qovery documentation where this is stated ? Advantages of picking one region over the other? permissions on GitHub?